examples/3d/load_gltf.rs

```diff
@@ -1,3 +1,5 @@
+//! Loads and renders a gltf file as a scene
```

Comment:
```suggestion
//! Loads and renders a glTF file as a scene.
```

This is the capitalization that Khronos [uses](https://www.khronos.org/gltf/). If one is referring to the file extension instead, they should write “`.gltf` file” instead. It is ok to say “glTF file” since it is another wording for “file with glTF format”.

I also added the period.